wearApp/src/main/java/dev/johnoreilly/confetti/wear/ConfettiApplication.kt

```diff
@@ -19,15 +20,25 @@ import org.koin.androidx.workmanager.koin.workManagerFactory
 
 class ConfettiApplication : Application(), ImageLoaderFactory {
 
+    private val isFirebaseInstalled
+        get() = try {
+            FirebaseApp.getInstance()
+            true
+        } catch (ise: IllegalStateException) {
+            false
+        }
+
     override fun newImageLoader(): ImageLoader = get()
 
     override fun onCreate() {
         super.onCreate()
 
-        if (!BuildConfig.DEBUG) {
-            Firebase.crashlytics.setCrashlyticsCollectionEnabled(true)
-            Firebase.crashlytics.setCustomKeys {
-                key("appName", "wearApp")
+        if (isFirebaseInstalled) {
```

Comment:
Just to be sure, Is this what you meant @yschimke?

Reply:
Yeah, I think so.  The Firebase apis won't work in robolectric at the moment, so you either need to check before, or catch the errors.